setup.py

```diff
@@ -38,7 +38,7 @@
     ],
     packages=find_packages(exclude=("tests",)),
     extras_require={
-        "dev": ["flake8", "black==22.3.0", "isort", "responses"],
+        "dev": ["flake8", "black==22.3.0", "isort", "responses","twine"],
```

Comment:
what is `twine`?

Reply:
oh got it :) I didn't know it about it, cool!

Reply:
Yes we use `twine` when publishing updates as documented here: https://roboflow.slab.com/posts/publishing-roboflow-pip-packages-k1a2z4ck